public/js/editors/libraries.js

```diff
@@ -688,7 +688,7 @@ var libraries = [
   }
 ];
 
-window.libraries = libraries; // expose a command line API
+window.libraries = _.sortBy(libraries, function(lib) { return lib.label; }); // expose a command line API
```

Comment:
Is underscore floating around in the globals somewhere?! (...I'm not even sure it's intentionally pulled in!)